guides/01_getting-started/03_sharing-your-app.md

```diff
@@ -464,3 +465,17 @@ demo.launch(max_file_size=5 * gr.FileSize.MB)
 ```
 
 Please make sure you are running the latest version of `gradio` for these security settings to apply.
+
```

Comment:
Personally think this is way too much content about analytics in the getting started guide @abidlabs

Reply:
This guide is no longer in the getting started section :)